lib/covDev.cu

```diff
@@ -140,12 +144,24 @@ namespace quda
 
     {
       constexpr int nDim = 4;
-      CovDevArg<Float, nColor, recon, nDim> arg(out, in, U, mu, parity, dagger, comm_override);
-      CovDev<decltype(arg)> covDev(arg, out, in);
 
-      dslash::DslashPolicyTune<decltype(covDev)> policy(
-        covDev, const_cast<cudaColorSpinorField *>(static_cast<const cudaColorSpinorField *>(&in)), in.VolumeCB(),
-        in.GhostFaceCB(), profile);
+      if (in.Nspin() == 4) {
```

Comment:
This needs to be updated since we no longer have `cudaColorSpinorField` objects anymore